packages/react-devtools-shared/src/backend/views/Highlighter/index.js

```diff
@@ -104,15 +104,21 @@ export default function setupHighlighter(
     const renderer = agent.rendererInterfaces[rendererID];
     if (renderer == null) {
```

Comment:
Why `==` and not `===`?

Reply:
To check if a value isn't equal to `undefined` or `null` with a single statement